src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/CrossTargetSubscriptionHostBase.cs

```diff
@@ -321,34 +321,14 @@ private void DisposeAndClearSubscriptions()
             }
         }
 
-        private JoinableTask<T> ExecuteWithinLockAsync<T>(Func<Task<T>> task)
+        private Task<T> ExecuteWithinLockAsync<T>(Func<Task<T>> task)
```

Comment:
>Task<T> [](start = 16, length = 7)

With this change, we now need a ConfigureAwait on lines 68, 76, 158, 230 and 302